### PR TITLE
update addon lts testing

### PR DESCRIPTION
**Pull request by kellyselden**

### CODE REVIEW

File: blueprints/addon/files/.travis.yml [hunk 1]
```diff
@@ -13,8 +13,8 @@ sudo: false
 <% } %>
 env:
   # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
-  - EMBER_TRY_SCENARIO=ember-lts-2.4
```
Comment (Turbo87) [inline blueprints/addon/files/.travis.yml]: do we want to remove 2.4 testing? that doesn't fit to comment above where I understand it as "testing all LTS's"

Comment (kellyselden) [inline blueprints/addon/files/.travis.yml]: I could have sworn it said last two LTS's. Must have changed recently?

Comment (Turbo87) [inline blueprints/addon/files/.travis.yml]: I don't know. I think @kategengler wanted to change this anyways to not actually use different Travis envs, so maybe we should wait for that to reduce blueprint churn?

/cc @rwjblue

Comment (stefanpenner) [inline blueprints/addon/files/.travis.yml]: lets update the comment (as discussed in today's meeting)

Comment (kellyselden) [inline blueprints/addon/files/.travis.yml]: Comment has been updated

**Comment by kellyselden** [comment]
I will wait for @Turbo87 approval because of concern over blueprint churn.

**Comment by Turbo87** [comment]
@homu r+

**Comment by Turbo87** [comment]
@kellyselden I think your original changes are in `beta` already, right? in that case we will need to cherry pick this down to the beta branch.

**Comment by Turbo87** [comment]
@homu retry

**Comment by kellyselden** [comment]
> I think your original changes are in beta already, right? in that case we will need to cherry pick this down to the beta branch.

@Turbo87 original changes?

**Comment by Turbo87** [comment]
> original changes?

@kellyselden nevermind... thought about something else 🙈

**Comment by Turbo87** [comment]
@homu r+

**Comment by Turbo87** [comment]
damn it!

@homu retry !!

**Pull request merged**